opsdroid/connector/matrix/connector.py

```diff
@@ -270,14 +270,12 @@ async def _parse_sync_response(self, response):
             ][0]
             sender = await self.get_nick(None, invite_event.sender)
 
-            await self.opsdroid.parse(
-                events.UserInvite(
-                    target=roomid,
-                    user_id=invite_event.sender,
-                    user=sender,
-                    connector=self,
-                    raw_event=invite_event,
-                )
+            yield events.UserInvite(
+                target=roomid,
+                user_id=invite_event.sender,
+                user=sender,
+                connector=self,
+                raw_event=invite_event,
```

Comment:
So just because I don't really understand `yield` that well: when does this then come back to the function and run the stuff below? Does the function resume from here next time it's called?

Reply:
correct, the function is suspended and resumed from below the yield on the next iteration.